fastlane/lanes/localization.rb

```diff
@@ -158,7 +158,7 @@
     )
   end
 
-  # Updates the metadata (from `fastlane/{metadata|jetpack_metadata}/android/*/*.txt`) in the PlayStore (Main store listing).
+  # Updates the metadata in the PlayStore (Main store listing) from the content of `fastlane/{metadata|jetpack_metadata}/android/*/*.txt` files
```

Comment:
Super nitpick:

```suggestion
  # Updates the metadata in the Play Store (Main store listing) from the content of `fastlane/{metadata|jetpack_metadata}/android/*/*.txt` files
```

Source:

<img width="599" alt="image" src="https://user-images.githubusercontent.com/1218433/162669277-674ab48e-10ba-46c6-8f51-01140ead03a3.png">